test/unit-tests/function/unit/toBest.test.js

```diff
@@ -0,0 +1,81 @@
+import math from '../../../../src/defaultInstance.js'
+import assert from 'assert'
+
+const Unit = math.create().Unit
+
+describe('toBest', function () {
+  it('should return the best unit without any parameters', function () {
+    assert.equal((new Unit(2 / 3, 'cm').toBest()).units[0].unit.name, 'm')
```

Comment:
Nice and thorough unit tests 👌.

Maybe it is handy to create a little helper function in this file to test the name, prefix, and value? Like `assertUnit(actualUnit, expectedName, expectedPrefix, expectedValue)`. What do you think?